scripts/rabbitmq-server-ha.ocf

```diff
@@ -66,6 +67,7 @@ OCF_RESKEY_rmq_feature_local_list_queues_default=true
 : ${OCF_RESKEY_mnesia_schema_base=${OCF_RESKEY_mnesia_schema_base_default}}
 : ${OCF_RESKEY_pid_file=${OCF_RESKEY_pid_file_default}}
 : ${OCF_RESKEY_node_port=${OCF_RESKEY_node_port_default}}
+: ${OCF_RESKEY_vhost=${OCF_RESKEY_vhost_default}}
```

Comment:
I suggest renaming it to `OCF_RESKEY_default_vhost` ( and `OCF_RESKEY_default_vhost_default` ) to match the config option name in `rabbitmq.config` . And mentioning somewhere in comments that using this option is mandatory for proper functioning of the script.